### ISSUE TITLE
PinebookPro unable to boot tow-boot installer.

### ISSUE THREAD

**Issue by Nik--** [Tow-Boot/Tow-Boot#131]
I have U-Boot flashed to my SPI and Tow-Boot image on my SD Card. When I try to boot from within u-boot, it seems to find the boot file within the SD card, but it fails to load, resulting in `No menu system available` message.
I have looked around for solutions and saw that erasing SPI might be a solution, but I am not sure that it would work. Given the risks involved into that, I prefer to ask if there is any other workaround for this problem.

Here is the full message log during boot:
<img src="https://user-images.githubusercontent.com/1042950/157192985-27315ea4-b7d4-4bac-aa1b-8883ef1871db.JPG" width=10% height=10%>

**Comment by benthetechguy** [comment]
The RK3399's hard-coded boot order is:
1. SPI
2. eMMC
3. SD Card

Normal U-Boot can't boot to the Tow-Boot SPI installer, getting that error.
Your only solution is to erase the SPI. What risks do you speak of? There are basically none.

**Comment by Nik--** [comment]
Well, reading the guide for flash erasing the Pinebook Pro, it looked like I've had to open the device and short circuit two pins to erase it. Therefore I thought that it was a risky thing to do. But apparently, there was another way, by installing `mtd-utils` package and running `flash_erase /dev/mtd0 0 0`, the SPI was erased. But for me that was still not enough, as the eMMC apparently had another u-boot in it, so I neutered it as well by running `dd if=/dev/zero of=/dev/mmcblk2 bs=512 seek=64 count=16`. In the end, the tow-boot installer managed to run and flash successfully to SPI.
